sql/core/src/main/scala/org/apache/spark/sql/execution/command/CommandUtils.scala

```diff
@@ -53,7 +53,11 @@ class PathFilterIgnoreNonData(stagingDir: String) extends PathFilter with Serial
 object CommandUtils extends Logging {
 
   /** Change statistics after changing data by commands. */
-  def updateTableStats(sparkSession: SparkSession, table: CatalogTable): Unit = {
+  def updateTableStats(
+      sparkSession: SparkSession,
+      table: CatalogTable,
+      partitionSpecs: Iterable[Map[String, Option[String]]] = Iterable.empty,
+      isDropPartition: Boolean = false): Unit = {
```

Comment:
can we add some comments to this method and its parameters? as it has became more complex than before now.

Reply:
removed the new parameters for the new approach